certification/internal/openshift/openshift_version.go

```diff
@@ -17,7 +17,7 @@ func GetOpenshiftClusterVersion() (runtime.OpenshiftClusterVersion, error) {
 	if _, ok := os.LookupEnv("KUBECONFIG"); !ok {
 		return runtime.UnknownOpenshiftClusterVersion(), fmt.Errorf("KUBECONFIG not specified")
 	}
-	kubeConfig, err := ctrl.GetConfig()
+	kubeConfig, err := ctrl.GetConfig() // TODO(): Identify how to get the config/rest client using an io.Reader or similar instead of this.
```

Comment:
Is this todo for a future pr?

Reply:
Yep. I didn't want to include it here just because it changed even more things than I'm already changing.

Reply:
That makes sense, and I see the larger TODO where you call this out explicitly.

Reply:
Resolving this.